src/Exporter.php

```diff
@@ -9,10 +9,10 @@
  * file that was distributed with this source code.
  */
 
-namespace Exporter;
+namespace Sonata\Exporter;
 
-use Exporter\Source\SourceIteratorInterface;
-use Exporter\Writer\TypedWriterInterface;
+use Sonata\Exporter\Source\SourceIteratorInterface;
+use Sonata\Exporter\Writer\TypedWriterInterface;
```

Comment:
you don't have to trigger class exists here?

Reply:
Good catch! It's the only one, I checked with `grep -L`

Reply:
Are you sure? I think I saw another one, will recheck later, on my phone now

Reply:
You're right, gonna retry that grep :(

Reply:
Ok this time I found many more…